oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/web/OAuth2TokenRevocationEndpointFilter.java

```diff
@@ -61,12 +59,15 @@
 	private static final String DEFAULT_TOKEN_REVOCATION_ENDPOINT_URI = "/oauth2/revoke";
 
 	private final AuthenticationManager authenticationManager;
+	private AuthenticationConverter tokenRevocationAuthenticationConverter;
```

Comment:
Rename `tokenRevocationAuthenticationConverter`, `authenticationSuccessHandler` and  `authenticationFailureHandler` to be consistent with the names in `OAuth2TokenRevocationEndpointConfigurer`